src/Components/Card/Card.jsx

```diff
@@ -1,35 +1,121 @@
-import React from 'react'
-import {FiHeart} from "react-icons/fi"
-import {FiMessageCircle} from "react-icons/fi"
-import {FiSend} from "react-icons/fi"
-import {BsBookmark} from "react-icons/bs"
-import {ImSearch} from "react-icons/im"
-import logo from "../../Assets/Images/logo.png"
-
-function Card() {
+import React, { useState } from "react";
+import { FiHeart } from "react-icons/fi";
+import { FiMessageCircle } from "react-icons/fi";
```

Comment:
instead of writing multiple lines of import, you could have imported "fiheart" and "fimessagecircle" in one line. Same applies to all the other components that are getting imported from same file